docs/pages/1-generate-https.md

```diff
@@ -168,205 +166,64 @@ this is pronounced ENGINE-X - it took me only 8 years to know that :P). As a san
 
 
 ## SSL Certificates
-We will be interacting with the nginx on the host, and following steps to:
-
- - start nginx (already done)
- - install tiny acme
- - generate certificates
- - using tinyacme to get them certified
- - moving them to where they need to be.
- - add a reminder or some other method to renew within 89 days
- 
-Once we do this, we will stop the local nginx, build our container, and bind the certificates to the server in our container when we start it.For the next set up steps, I'll
-walk through them manually, and if you choose, you could put them into a script.
-The certificates need to be renewed every 89 days, so you will need to have some
-strategy in place. It's likely the case that your server won't be running that long,
-so it doesn't matter :) If you are doing this for the first time, start at step 1. If you
-are refreshing your certificates, jump down to step 4.
-
-
-### Step 1. Define Registration Details
-Let's define some environment variables that will go into our configuration. You need to define your email, the domain (e.g, expfactory.org) the state, and county. Here is an 
-example:
-
-```bash
-EMAIL=dinosaur@planetearth.com
-DOMAIN=expfactory.dynu.net
-STATE=California
-COUNTY=”San Mateo County”
-```
+We'll use "certbot" to install and renew certificates.
 
-Let's work in our $HOME directory to generate a `csr_details.txt` file that we will use
-to issue the request for credentials.
+### Step 1. Install certbot
 
-```bash
-cd $HOME
-```
-Then, write the csr_details.txt file
-
-```bash
-cat > csr_details.txt <<-EOF
-[req]
-default_bits = 2048
-prompt = no
-default_md = sha256
-req_extensions = req_ext
-distinguished_name = dn
- 
-[ dn ]
-C=US
-ST=$STATE
-L=$COUNTY
-O=End Point
-OU=$DOMAIN
-emailAddress=$EMAIL
-CN = www.$DOMAIN
- 
-[ req_ext ]
-subjectAltName = @alt_names
- 
-[ alt_names ]
-DNS.1 = $DOMAIN
-DNS.2 = www.$DOMAIN
-EOF
-```
-
-This should produce a file `csr_details.txt` in your present working directory
-that we will use later.
-
-
-### Step 2. Install Acme Tiny
-We are going to use [Acme Tiny](https://github.com/diafygi/acme-tiny) to both issue
-and renvew our certificates. Let's install it first. I chose to clone to `/tmp` and
-install to `/opt`, you can obviously mix this up. If you are using a Droplet (or
-a server without python) make sure to install it first:
+Certbot automates certificate generation and renewal. In other words, it makes it really easy to setup SSL.
 
-```bash
-# Python
-sudo apt-get install -y python
 ```
-
-Then install Acme Tiny.
-
-```bash
-sudo mkdir /opt/acme_tiny
-cd /tmp && git clone https://github.com/diafygi/acme-tiny
-sudo mv acme-tiny /opt/acme-tiny/
-sudo chown $USER -R /opt/acme-tiny
+sudo add-apt-repository ppa:certbot/certbot
+sudo apt-get update
+sudo apt-get install python-certbot-nginx
 ```
```

Comment:
Just a sanity check - do we not need to install python (e.g., does certbot (which I think based on the name needs python?) ) install it as a dependency? I needed to install python for Acme Tiny (the server doesn't come with it)